### ISSUE TITLE
`ebay-tab`: manual activation doesn't seem to work

### ISSUE THREAD

**Issue by darkwebdev** [eBay/ebayui-core#887]
# Bug Report

## Description

1. go to [marketplace](http://app.uimarketplace.svc.32.tess.io/?tag=ebay-tab)
2. scroll to the bottom to see the `activation="manual"` example
3. click any tab, press left/right arrow to move focus to another tab
4. press `space` or `enter`
5. =(

## Workaround

Possible cause:
https://github.com/eBay/ebayui-core/blob/c019071ef7925e66186ee4b0eff68b3ccaa3c87f/src/components/ebay-tab/index.js#L40

in the function that handles `space`/`enter` presses:
```
eventUtils.handleActionKeydown(event, () => {
            event.preventDefault();

            if (this.state.activation === 'auto') {
                this.setIndex(el.dataset.index);
            }
});
```
condition is mistakenly inverted and should be
```this.state.activation === 'manual'```

**Comment by ianmcburnie** [comment]
@darkwebdev thanks for reporting.

@agliga let's investigate and try and get a fix out this week for 4.2.0-1.

**Comment by ianmcburnie** [comment]
Fixed for v4.2.0
